src/webvtt/data.rs

```diff
@@ -406,15 +406,25 @@ impl TimedEventInterface for WebVttCue {
 
 impl Display for WebVttCue {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(
+        writeln!(f)?;
+
+        match &self.identifier {
+            Some(id) if !id.is_empty() => {
+                writeln!(f, "{}", id)?;
+            }
+            _ => {}
+        }
```

Comment:
The bug here is caused by an issue with the parsing logic, so I think it'd be better to fix the problem at its root by updating the parsing logic in `src/webvtt/parse.rs` at line 77:

```
pair(opt(space0), parse_cue_timing),
```

Using `space1` instead of `space0` should resolve this.

Combined with the above change, should make it unnecessary to modify this implementation

Reply:
Done ✅